README.md

```diff
@@ -70,68 +70,46 @@ or a function with zero or one arity which returns one. If it is a function, it
 evaluated for every request. If the function is arity one, the `Conn` struct will be
 passed to it, in order to have more flexibility in dynamic routing.
 
-### Usage in Phoenix
-The Phoenix default autogenerated project assumes that you'll want to
-parse all request bodies coming to your Phoenix server and puts `Plug.Parsers`
-directly in your `endpoint.ex`. If you're using something like ReverseProxyPlug,
-this is likely not what you want — in this case you'll want to move Plug.Parsers
-out of your endpoint and into specific router pipelines or routes themselves.
+### Modifying the client request body
+
+You can modify various aspects of the client request by simply modifying the
+`Conn` struct. In case you want to modify the request body, fetch it using
+`Conn.read_body/2`, make your changes, and leave it under
+`Conn.assigns[:raw_body]`. ReverseProxyPlug will use that as the request body.
+In case a custom raw body is not present, ReverseProxyPlug will fetch it from
+the `Conn` struct directly.
+
+## Configuration options
```

Comment:
I moved all options under this new umbrella heading. Level 2 `##` so that it appears on the sidebar in Hexdocs.pm.